website/docs/installation/fonts.mdx

```diff
@@ -13,6 +16,15 @@ To see the icons displayed in Oh My Posh, **install** a [Nerd Font][nerdfonts],
 
 #### Installation
 
+<Tabs
+    defaultValue="oh-my-posh"
+    values={[
+        { label: 'oh-my-posh', value: 'oh-my-posh' },
```

Comment:
```suggestion
        { label: 'oh-my-posh', value: 'Oh My Posh' },
```

Reply:
Shouldn't the `label` be the one changed?

Reply:
You are correct!